test/distributed/fsdp/test_fsdp_mixed_precision.py

```diff
@@ -756,5 +756,47 @@ def test_mixed_precision_e2e_full_shard(self):
 
 instantiate_parametrized_tests(TestFSDPMixedPrecisionSharded)
 
+
+class IgnoredModule(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.l = nn.Linear(100, 100)
+
+    def forward(self, x):
+        return self.l(x)
+
+
+class Model(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.l1 = nn.Linear(100, 100)
+        self.ignored = IgnoredModule()
+        self.l2 = nn.Linear(100, 100)
+
+    def forward(self, x):
+        return self.l2(self.ignored(self.l1(x)))
+
+
+class TestFSDPMixedPrecisionIgnoredModules(FSDPTest):
+    @property
+    def world_size(self):
+        return 1
+
+    @skip_if_lt_x_gpu(1)
+    def test_mixed_precision_with_ignored_module(self):
+        model = Model().cuda()
+        bfloat16 = MixedPrecision(param_dtype=torch.bfloat16)
```

Comment:
As a heads up, you might want to use `float16` here if it also reproduces the error since otherwise you need to handle conditional BF16 support in CI:
https://github.com/pytorch/pytorch/blob/f99818fbc52b90851a3ef2af729901332ea11289/test/distributed/fsdp/test_fsdp_mixed_precision.py#L81-L83

Reply:
(Assuming that this test is meant to be landed)

Reply:
ah, good point, let me update.